subset.go

```diff
@@ -46,6 +48,14 @@ func checkSubset(expected, target reflect.Value, visited map[uintptr]*visit, dep
 		// fmt.Println("!target.IsValid()")
 		return false
 	}
+
+	if (expected.Kind() == reflect.Int) && (target.Kind() == reflect.Float64) {
```

Comment:
nit: Can you add a note here that we modified this to enabled number comparison?

Reply:
done 👍